io.github.FailurePoint.RandomNumberFive.json

```diff
@@ -0,0 +1,38 @@
+{
+    "app-id" : "io.github.FailurePoint.RandomNumberFive",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "random-number-five",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "random-number-five",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "file:///home/ace/Projects"
```

Comment:
it won't find the source here.

Reply:
It has been fixed. Do I need to submit a new PR with the fix in it?

Reply:
Just push it here.